<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MassDvplTool</class>
 <widget class="QMainWindow" name="MassDvplTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>208</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>453</width>
    <height>208</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>453</width>
    <height>208</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>(Un)Pack DVPL</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="tb_SrcPath">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_SrcPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dvpl</string>
    </property>
    <property name="buddy">
     <cstring>tb_SrcPath</cstring>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_DstPath">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_DstPath">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Txt</string>
    </property>
    <property name="buddy">
     <cstring>tb_DstPath</cstring>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_SrcChange">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_DstChange">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>130</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Обзор</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="rb_Pack">
           <property name="text">
            <string>Запаковать</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rb_Unpack">
           <property name="text">
            <string>Распаковать</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Start">
         <property name="text">
          <string>Начать</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_Stop">
         <property name="text">
          <string>Остановить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="QProgressBar" name="status">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>453</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menu_ChangePath">
   <property name="text">
    <string>Выбрать путь</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
